test/types/scalar_test.rb

```diff
@@ -38,6 +38,22 @@ class ScalarTest < ActiveSupport::TestCase
     assert_equal false, boolean.value
   end
 
+  test "boolean casting" do
+    boolean = Kredis.boolean "myscalar"
+
+    boolean.value = true
+    assert_equal "1", boolean.get
+
+    boolean.value = false
+    assert_equal "0", boolean.get
+
+    boolean.set "true"
```

Comment:
Is it neccessary on this test case?

Reply:
I’m testing backwards compatibility here so I think it’s necessary.

Reply:
I see